core/sodasql/cli/cli.py

```diff
@@ -31,7 +31,10 @@
 
 LoggingHelper.configure_for_cli()
 logger = logging.getLogger(__name__)
-
+latest_version = luddite.get_version_pypi('soda-sql')
```

Comment:
There is no soda-sql anymore, it should be soda-sql-core. https://pypi.org/project/soda-sql-core/

Reply:
This only warns when CLI is invoked, the version check should also happen for programmatic scans using ScanBuilder.

Reply:
Added a __call__ method to ScanBuilder for this purpose. Will add the method to cli_helper class once the other branch is merged.

Reply:
Yes for the commit, please check it out (I can see the changes in the branch)

Tested offline in my computer and worked fine, maybe because of some cache? I will add an exception to this right now.